qemu/tests/cfg/nvdimm.cfg

```diff
@@ -38,6 +38,7 @@
     size_mem = 1G
     share_mem = yes
     nv_file = "${mount_dir}/nv0"
+    depends_pkgs = "ndctl"
```

Comment:
I think should add a space, like "ndctl ", otherwise in other cases, the parameter will become "ndctlndctl-devel daxctl-devel pandoc"

Reply:
Updated.

Reply:
A good catch ~   Generally, i prefer add a space when use `+=`,  but the current change is ok for me too, thanks for raise it :)